src/components/Checkbox/Checkbox.vue

```diff
@@ -0,0 +1,169 @@
+<!--
+  -
+  - @author Julius Härtl <jus@bitgrid.net>
+  - @author Greta Doci <gretadoci@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+<docs>
+## One checkbox with a hint
+
+```vue
+<template>
+		<Checkbox
+			label="some text"
+			v-model="checkedTarget"/>
+</template>
+```
+</docs>
+
+<template>
+	<label>
+		<input
+			:id="id"
+			type="checkbox"
+			:checked="value"
+			:disabled="disabled"
+			@keydown.enter.exact.prevent="checkInput"
+			@change="onChange">
+		{{ label }}
+	</label>
+</template>
+
+<script>
+import GenRandomId from '../../utils/GenRandomId'
+export default {
+	name: 'Checkbox',
+	props: {
+		/**
+		 * id attribute of the checkbox element
+		 */
+		id: {
+			type: String,
+			default: () => 'checkbox-' + GenRandomId,
+
+		},
+		label: {
+			type: String,
+			required: true,
+		},
+		/**
+		 * value of the checkbox input
+		 */
+		value: {
+			type: Boolean,
+			default: false,
```

Comment:
```suggestion
			required: true,
```

Or am I missing something? :)